tests/python/gaia-ui-tests/gaiatest/apps/phone/regions/keypad.py

```diff
@@ -74,10 +74,10 @@ class Keypad(BaseKeypad, Phone):
 
     def __init__(self, marionette):
         Phone.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
+        self.wait_to_be_displayed()
         self.apps.switch_to_displayed_app()
         keypad_toolbar_button = self.marionette.find_element(*self._keypad_toolbar_button_locator)
-        Wait(self.marionette).until(lambda m: 'toolbar-option-selected' in keypad_toolbar_button.get_attribute('class'))
+        self.wait_for_condition(lambda m: 'toolbar-option-selected' in keypad_toolbar_button.get_attribute('class'))
```

Comment:
Please don't do this, this causes a failure in test_call_log_all_calls.py because wait_for_condition doesn't exist on the Base class anymore.
I don't see why this change was necessary anyhow.